XenModel/Messages.resx

```diff
@@ -6524,6 +6524,36 @@ This might result in failure to migrate VMs to this server during the RPU or to
   <data name="HOTFIX_CHECK" xml:space="preserve">
     <value>Hotfix check</value>
   </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_DESCRIPTION_CU" xml:space="preserve">
+    <value>A new CU is available on {0}. After {1} there will be no further hotfixes released on {2}. Please update to the latest Cumulative Update.</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_DESCRIPTION_EOL" xml:space="preserve">
+    <value>{0} has reached End of Life on {1}. There will be no further hotfixes released after this date. Please upgrade to the latest supported version.</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_DESCRIPTION_EOL_FREE" xml:space="preserve">
+    <value>{0} has reached End of Life on {1}. There will be no further hotfixes released after this date. Please upgrade to the latest CR.</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_DESCRIPTION_FREE" xml:space="preserve">
+    <value>{0} has reached End of Life for express customers on {1}. You are no longer eligible for new hotfixes after this date. Please upgrade to the latest CR.</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_TITLE_CU" xml:space="preserve">
+    <value>A new Cumulative Update is available</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_TITLE_EOL" xml:space="preserve">
+    <value>{0} has reached End of Life</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_ALERT_TITLE_FREE" xml:space="preserve">
+    <value>{0} has reached End of Life for express customers</value>
+  </data>
+  <data name="HOTFIX_ELIGIBILITY_WARNING_CU" xml:space="preserve">
+    <value>Not eligible for new hotfixes after {0}</value>

```

Comment:
I think this is missing `released`

Reply:
This might need to be reworded; It is meant to warn that there will be no further hotfixes released on the current version after the specified date, as there is a CU available and the users should basically get the CU.

Reply:
`No further hotfixes released after {0}`.